docs/tutorials/join-dal-baker/prepare-account.md

```diff
@@ -34,81 +57,46 @@ This command creates an account and associates it with the `my_baker` alias:
    octez-client show address my_baker
    ```
 
-1. Record this address in a shell variable so you can use it for commands that cannot get addresses by their Octez client aliases:
-
-   ```bash
-   MY_BAKER="$(octez-client show address my_baker | head -n 1 | cut -d ' ' -f 2)"
-   ```
-
    At this point, the balance of the `my_baker` account is still zero, as you can see by running this command:
 
    ```bash
    octez-client get balance for my_baker
    ```
 
-   If you get an error on this command, your local node isn't ready yet.
-   Until the node has finished bootstrapping, pass the public RPC endpoint for Weeklynet in the `--endpoint` argument, as in this example:
-
-   ```bash
-   octez-client --endpoint https://rpc.weeklynet-2024-01-17.teztnets.com get balance for my_baker
-   ```
-
-   Don't set the client endpoint to the public node permanently because it should use your local node whenever possible.
-   For bakers, it's important to set the Octez client to use their node rather than a public node because the baker daemon uses the client configuration and the baker daemon should use the local node.
+1. Get at least 6,000 tez from the Ghostnet faucet.
 
-1. Get some tez from the Weeklynet faucet.
+   The account must stake tez to get consensus and DAL rights.
+   To get tez, use the Ghostnet faucet linked from https://teztnets.com/ghostnet-about to send tez to the baker account.
 
-   In order to get some consensus and DAL rights, we need to put some tez in the account. Fortunately, getting free testnet tez is easy thanks to the testnet faucet. To use it, we need to enter the generated address in the Weeklynet faucet linked from https://teztnets.com/weeklynet-about. We need at least 6k tez for running a baker but the more tez we have the more rights we will get and the shorter we will have to wait to produce blocks and attestations. That being said, baking with too much stake prevents us from leaving the network without disturbing or even halting it so to avoid breaking the network for all other testers let's not be too greedy. 50k tez is enough to get enough rights to easily check if our baker behaves as expected while not disturbing the network too much when our baker stops operating.
+   Running a baker requires staking at least 6,000 tez, but the more tez it stakes, the more rights it gets and the lass time it has to wait to produce blocks and make attestations.
+   However, baking with too large of a stake can cause problems when the baker stops because it holds a significant portion of baking rights.
```

Comment:
Is this paragraph important for this tutorial? I don't think any new baker will get a sufficient large portion of the stake. 

It sounds purely informative, if we want to keep it, maybe we could use an informative block?

Reply:
Yeah, this is not relevant for Ghostnet, I'll remove it.